index.js

```diff
@@ -68,7 +68,7 @@ function SMTPServer(hostname, options) {
         honorCipherOrder: options && "honorCipherOrder" in options ? options.honorCipherOrder : true,
         requestOCSP: options && "requestOCSP" in options ? options.requestOCSP : false,
         key: options.key,
```

Comment:
For consistency and safety I think using
`options && options.key` here
is better than providing a default of an empty object
since someone could still explicitly pass null or undefined.

Reply:
I keep getting this error I think line 70 should be 
`key: options && options.key`


since without this we get the following error

```
TypeError: Cannot read properties of undefined (reading 'key')
    at new SMTPServer (/Users/rfk851/Development/Repositories/Architecture-Sandbox/nodejs-sandbox/smtp-receiver-connector-proj/node_modules/smtp-receiver/index.js:71:22)

```

Reply:
I keep getting this error I think line 70 and 71 should be
```
key: options && options.key
cert: options && options.cert
```
since without this we get the following error

```
TypeError: Cannot read properties of undefined (reading 'key')
    at new SMTPServer (/Users/rfk851/Development/Repositories/Architecture-Sandbox/nodejs-sandbox/smtp-receiver-connector-proj/node_modules/smtp-receiver/index.js:71:22)
```